Diagnostics/Get-NetView.PS1

```diff
@@ -1618,6 +1634,7 @@ function Worker {
     EnvDestroy        -OutDir $workDir
     EnvCreate         -OutDir $workDir
 
+    Version           -OutDir $workDir
```

Comment:
Version output here means, its not visible on screen as the data scrolls by.  You should take the contents of Version() and execute them from inside Completion().  Such that it displays to screen at end of execution and to the output file you created.